lib/matplotlib/path.py

```diff
@@ -558,14 +558,12 @@ def intersects_bbox(self, bbox, filled=True):
         :class:`~matplotlib.transforms.Bbox`.
 
         *filled*, when True, treats the path as if it was filled.
-        That is, if one path completely encloses the other,
-        :meth:`intersects_path` will return True.
+        That is, if the path completely encloses the bounding box,
+        :meth:`intersects_bbox` will return True.
+
+        The bounding box is always considered filled.
         """
-        from .transforms import BboxTransformTo
-        rectangle = self.unit_rectangle().transformed(
```

Comment:
No, I checked that. This is just a somewhat confusing way of getting a rectangle that represents the bounding box.